go.sum

```diff
@@ -340,7 +340,6 @@ modernc.org/golex v1.0.0/go.mod h1:b/QX9oBD/LhixY6NDh+IdGv17hgB+51fET1i2kPSmvk=
 modernc.org/mathutil v1.0.0/go.mod h1:wU0vUrJsVWBZ4P6e7xtFJEhFSNsfRLJ8H458uRjg03k=
 modernc.org/strutil v1.0.0/go.mod h1:lstksw84oURvj9y3tn8lGvRxyRC1S2+g5uuIzNfIOBs=
 modernc.org/xc v1.0.0/go.mod h1:mRNCo0bvLjGhHO9WsyuKVU4q0ceiDDDoEeWDJHrNx8I=
-sigs.k8s.io/cluster-api v0.2.5 h1:N3nz2U62tE1lm4szCrEDMPMg/8RMiLaYgHWZJqkSuLE=
 sigs.k8s.io/cluster-api v0.2.5/go.mod h1:Agc72Ra5LMOkQQ2v/Ywv1KUemaYAwvkQ+G59Ym5H8e4=
```

Comment:
This line should also get removed with `make module`

Reply:
No - cabpk still uses v0.2.5 (we chatted on Slack about this)

Reply:
ah gotcha, sounds good

Reply:
ran `make modules`, no diff FYI